web/modules/custom/hoeringsportal_audit_log/src/EventSubscriber/ControllerListener.php

```diff
@@ -40,10 +40,21 @@ public function __construct(
    */
   public function onController(ControllerEvent $event): void {
     $pathInfo = $event->getRequest()->getPathInfo();
+    $requestUri = $event->getRequest()->getRequestUri();
+    $urlPatterns = $this->configHelper->getUrlPattern();
+
+    if ($urlPatterns) {
+      foreach ($urlPatterns as $urlPattern) {
+        if (@preg_match($urlPattern, $requestUri)) {
+          $this->logAuditMessage($pathInfo);
+          return;
+        }
+      }
+    }
```

Comment:
As per comment on `getUrlPattern(s)`, `$urlPatterns` is always an array (it may be empty):

```suggestion
    foreach ($urlPatterns as $urlPattern) {
      if (@preg_match($urlPattern, $requestUri)) {
        $this->logAuditMessage($pathInfo);
        return;
      }
    }
```